packages/components/snackbar/src/snackbar.ts

```diff
@@ -12,61 +13,22 @@ export interface SnackbarAction {
 
 export interface PuikSnackbarOptions {
   text: string
```

Comment:
not required before (you think it should be ?)

Reply:
A snackbar without text isn't usefull i think x)